main.py

```diff
@@ -140,12 +138,11 @@ def run_scenario(current_scenario):
     current_scenario.federated_computation_time_sec = end - start
 
     for method in current_scenario.methods:
-        print(method)
+        logger.info(f"{method}")
```

Comment:
Maybe this one is not required as it is logged just after?

Reply:
Agreed. Actually I changed it in PR #131